sys/include/net/gnrc/sixlowpan/ctx.h

```diff
@@ -94,6 +95,19 @@ gnrc_sixlowpan_ctx_t *gnrc_sixlowpan_ctx_lookup_addr(const ipv6_addr_t *addr);
  * @return  NULL if there is no such context.
  */
 gnrc_sixlowpan_ctx_t *gnrc_sixlowpan_ctx_lookup_id(uint8_t id);
+#else
+static inline gnrc_sixlowpan_ctx_t *gnrc_sixlowpan_ctx_lookup_addr(const ipv6_addr_t *addr)
+{
+    (void)addr;
+    return NULL;
+}
+
+static inline gnrc_sixlowpan_ctx_t *gnrc_sixlowpan_ctx_lookup_id(uint8_t id)
+{
+    (void)id;
+    return NULL;
+}
+#endif /* IS_USED(MODULE_GNRC_SIXLOWPAN_CTX) */
```

Comment:
Do you think the absence of those functions in case of `MODULE_GNRC_SIXLOWPAN_NO_CTX` would be cleaner handled like this?
```C
ctx = IS_USED(MODULE_GNRC_SIXLOWPAN_NO_CTX) ? NULL
      : gnrc_sixlowpan_ctx_lookup_addr(&unicast_prefix);
```

Reply:
No, then you have to touch every call site and inflict some ugliness on it.
A `static inline` no-op is much cleaner IMHO.